modules/ossm-security-mtls.adoc

```diff
@@ -52,16 +51,16 @@ Create a destination rule to configure {ProductShortName} to use mTLS when sendi
 
 [source,yaml]
 ----
-apiVersion: "networking.istio.io/v1alpha3"
-kind: "DestinationRule"
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
 metadata:
-  name: "default"
-  namespace: <CONTROL_PLANE_NAMESPACE>
+  name: default
+  namespace: <namespace>
 spec:
-  host: "*.local"
+  host: “*.<namespace>.svc.cluster.local”
```

Comment:
Are these quotes correct?

Reply:
Yeah, but the quote character is not the correct one. It's “ instead of ". If you try to create this DestinationRule, it is rejected: 
```
admission webhook "validation.istio.io" denied the request: configuration is invalid: domain name "“*.istio-system.svc.cluster.local”" invalid (label "“*" invalid)
```

Reply:
I fixed these quotes just now.